sagemaker-keyspaces/SageMaker_Keyspaces_ml_example.ipynb

```diff
@@ -0,0 +1,469 @@
+{
```

Comment:
<p>Style: remove the odd extra spaces in the markdown of this cell.</p><p> </p><p>Branding: revise all instances of <code>Sagemaker</code> to <code>SageMaker</code>	throughout the notebook.</p><p> </p><blockquote>Next, we will read the ingested data into Sagemaker and do feature engineering.</blockquote><p>Which SageMaker features are being used? I don't see any. Why is this notebook useful to SageMaker customers? I think the value-add within SageMaker/ML can be made more clear and compelling.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3166/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<ol><li>Changed <code>Sagemaker</code> to <code>SageMaker</code> .</li><li>Added more details to indicate what benefits this would provide to customers. </li></ol><p> </p>